providers/http.go

```diff
@@ -63,3 +63,72 @@ func CopyGRPCHeadersClientInterceptorFxOption() fx.Option {
 //
 // Consider using CopyGRPCHeadersClientInterceptorFxOption if you only want to provide it.
 var CopyGRPCHeadersClientInterceptor = client.CopyGRPCHeadersClientInterceptor
+
+// CopyGRPCHeadersHTTPClientInterceptorFxOption copies filtered Headers found in the Incoming GRPC metadata.MD to the Outgoing HTTP Request Headers.
+//
+// This is useful if you want to propagate them to the next service when using `http.Client`
+//
+// For Example: "authorization" header containing user token
+func CopyGRPCHeadersHTTPClientInterceptorFxOption() fx.Option {
+	return fx.Provide(
+		fx.Annotated{
+			Group:  groups.RESTClientInterceptors,
+			Target: client.CopyGRPCHeadersHTTPClientInterceptor,
```

Comment:
Shouldn't be REST here?

Reply:
No, it copies values from grpc incoming metadata.MD to external Headers (REST)

Reply:
ok